Source/Core/Core/IOS/VersionInfo.h

```diff
@@ -37,5 +37,30 @@ struct MemoryValues
 };
 
 const std::array<MemoryValues, 41>& GetMemoryValues();
+
+enum Feature
```

Comment:
I had actually made `Feature` an enum class at first, but having to implement `operator|` to get that to work seemed a bit weird... Didn't know that's how it's done in the STL!

Since operator| is the only one that is used, is it fine to implement just that one or should I make it satisfy BitmaskType?